frontend/initial_props.py

```diff
@@ -65,6 +66,7 @@ def create_initial_props_for_lambda(
             "debug": settings.DEBUG,
             "facebookAppId": settings.FACEBOOK_APP_ID,
             "nycGeoSearchOrigin": settings.NYC_GEOSEARCH_ORIGIN,
+            "contentfulCommonStrings": contentful.get_common_strings(),
```

Comment:
This could be bad, if contentful ever goes down.  We don't cache anything when there's a network error, so if cdn.contentful.com happens to timeout, all requests to our server could take an unusually long time, effectively resulting in a denial of service.

That said, contentful _does_ use a CDN so this request should be pretty darn fast and timeouts should be _extremely_ rare.  Lots of businesses depend on Contentful's CDN to be responsive so we could always just give this a shot and revisit if anything ever goes wrong.

Reply:
> Cool! My understanding is that the org site already has infrastructure to pull from contentful, so that one would be easier to change over. So that would just leave us with WOW still pulling from react-common, and presumably we _could_ do something similar with WOW.

Right, though there are a few wrinkles--more details at https://github.com/JustFixNYC/who-owns-what/pull/482!